libraries/botbuilder-dialogs-adaptive/src/actions/httpRequest.ts

```diff
@@ -5,7 +5,8 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
-import fetch from 'node-fetch';
+import { StatusCodes } from 'botbuilder-core';
+import fetch, { FetchError } from 'node-fetch';
```

Comment:
is the FetchError a type?

nit: consider using type { FetchError } from.
or something like that, some cases in the code does that, and performs faster

Reply:
I tried to import it as type but I get the following error. Should I import it before the if statement?
<img width="832" alt="image" src="https://user-images.githubusercontent.com/64815358/181834095-5fd8ccb2-028a-4cb1-a285-fb10de1565a7.png">

Reply:
ohh, right, its ok then, it has to be a class, nvm disregard my comment